contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -135,16 +108,14 @@ contract UniswapAnchoredView is AggregatorValidatorInterface, UniswapConfig, Own
      */
     function getUnderlyingPrice(address cToken) external view returns (uint) {
         TokenConfig memory config = getTokenConfigByCToken(cToken);
-         // Comptroller needs prices in the format: ${raw price} * 1e36 / baseUnit
-         // The baseUnit of an asset is the amount of the smallest denomination of that asset per whole.
-         // For example, the baseUnit of ETH is 1e18.
-         // Since the prices in this view have 6 decimals, we must scale them by 1e(36 - 6)/baseUnit
+         // Comptroller needs prices in the format: ${raw price} * 1e(36 - baseUnit)
+         // Since the prices in this view have 6 decimals, we must scale them by 1e(36 - 6 - baseUnit)
```

Comment:
Can we revert this to using the old comment?

We changed because the math in it was slightly off.